.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: node_js
-node_js: "6"
+os:
+  - linux
+  - osx
+node_js:
+  - "node"
```

Comment:
I thought about adding also build for node v4 cause some still use it, but then decided against it. People using node should update to the latest LTS at least.